sep-template.md

```diff
@@ -14,6 +14,11 @@ Discussion: <link to where discussion for this SEP is taking place, typically th
 "If you can't explain it simply, you don't understand it well enough." Provide a simplified and
 layman-accessible explanation of the SEP.
 
+## Dependencies
+Should clearly list the dependencies this SEP has on other SEPs or CAPs. May
+go into further discussion about the circumstances of the dependency,
+although a list sufficient.
+
```

Comment:
nit: formatting

```suggestion
## Dependencies

Should clearly list the dependencies this SEP has on other SEPs or CAPs. May go into further discussion about the circumstances of the dependency, although a list sufficient.
```

Reply:
I committed this, then removed it again, because the new line at the end is needed, and in the template there are no new lines under headings so this would be inconsistent.